archiver/src/main/java/com/netflix/conductor/archiver/job/QueueMessageJob.java

```diff
@@ -0,0 +1,59 @@
+package com.netflix.conductor.archiver.job;
+
+import com.netflix.conductor.archiver.config.AppConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.commons.collections.CollectionUtils.isNotEmpty;
+
+public class QueueMessageJob extends AbstractJob {
+    private static final Logger logger = LogManager.getLogger(QueueMessageJob.class);
+
+    public QueueMessageJob(HikariDataSource dataSource) {
+        super(dataSource);
+    }
+
+    @Override
+    public void cleanup() {
+        logger.info("Starting QueueMessageJob");
+        try {
+            AppConfig config = AppConfig.getInstance();
+            String cleanupWorkflows = config.cleanupMessageWorkflows();
+            String QUERY = "select id from queue_message qm\n" +
```

Comment:
Let's get rid of `\n` and replace by ` ` (space) so the final query in pg stats will be one line